src/Collector.cs

```diff
@@ -127,19 +142,30 @@ private void SendQueuedMessages(Socket socket)
         }
     }
 
-    private void Authenticate(Socket socket)
+    private bool Authenticate(Socket socket)
     {
-      var data = System.Text.Encoding.ASCII.GetBytes("hello version 1.0\n");
+      var data = System.Text.Encoding.ASCII.GetBytes("hello version dotnet/0.2.0\n");
```

Comment:
I have not found a good way to pull the version out of the assembly without including System.Reflection, nor to use a constant from the class in the assembly info.

I'll extract it and do... something.